cta_names_mappings.json

```diff
@@ -744,8 +744,6 @@
   },
   "NoName057(16)": {
   },
-  "PureCoder": {
```

Comment:
@KadeMorton - just wanted to confirm if removing this is intentional (as I don't see the corresponding change / removal in the md file)?